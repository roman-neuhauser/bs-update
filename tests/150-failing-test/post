. ../post.functions

trap cleanup EXIT

assert_fs -f "test/exit.actual"
assert_fs -f "test/err.actual"
assert_fs -f "test/out.actual"
assert_fs -f "test/exit.diff"
assert_fs -f "test/err.diff"
assert_fs -f "test/out.diff"
assert_same "expected/exit.diff" "test/exit.diff"
assert_same "expected/err.diff" "test/err.diff"
assert_same "expected/out.diff" "test/out.diff"

