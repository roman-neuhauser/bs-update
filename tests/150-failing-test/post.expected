good: -f test/exit.actual
good: -f test/err.actual
good: -f test/out.actual
good: -f test/exit.diff
good: -f test/err.diff
good: -f test/out.diff
